web/src/message_view.ts

```diff
@@ -880,6 +877,7 @@ export function fast_track_current_msg_list_to_anchor(anchor: string): void {
                 return msg_list_data.first()!.id;
             },
         });
+        message_lists.current.unfocus_navigation_bar();
```

Comment:
I'm not sure I understand some of these changes in when we do the checks for updating these.

Reply:
Yeah, we can make the focus / unfocus logic simple in the codebase but we need to nail down the experience we want here too.